src/tests/t_tokenize.c

```diff
@@ -35,12 +35,13 @@ struct test_case_type test_case_1 = {
 // Acronims and emojis.
 struct test_case_type test_case_2 = {
 	"We will play 🥎 with I.B.M.",
-	{ "we", "will", "play", "🥎",  "with", "i.b.m", NULL }
+	// { "we", "will", "play", "🥎",  "with", "i.b.m", NULL }
```

Comment:
Hmm.. maybe no emojis would be a good default. We probably want to parametrize this later.

Reply:
I think we could merge this, as Im working on the emoji, but it may take some more time.

I was thinking of going in way of translating, 🥎 => tennis; which is kinda the way ES does. see: https://unicode.org/emoji/charts/emoji-list.html

Reply:
Sounds good